zipkin-autoconfigure/ui/src/main/java/zipkin2/autoconfigure/ui/CompressionProperties.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.autoconfigure.ui;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.web.server.Compression;
+
+@ConfigurationProperties("server")
+class CompressionProperties {
+  public Compression getCompression() {
```

Comment:
Not a big deal but I presume spring already exposes a bean with this

Reply:
surprisingly no.. I tried :)